bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/FragmentContainerActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.livefront.bridgesample.scenario.activity
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.os.Parcelable
+import android.support.annotation.StringRes
+import android.support.v4.app.Fragment
+import android.view.MenuItem
+import com.livefront.bridgesample.R
+import com.livefront.bridgesample.base.BridgeBaseActivity
+import com.livefront.bridgesample.scenario.activity.FragmentContainerActivity.Companion.getNavigationIntent
+import com.livefront.bridgesample.util.handleHomeAsBack
+import com.livefront.bridgesample.util.setHomeAsUpToolbar
+import kotlinx.android.parcel.Parcelize
+import kotlinx.android.synthetic.main.basic_toolbar.toolbar
+
+/**
+ * A simple [Activity] with a container in which an initial [Fragment] may be placed. This is
+ * done by supplying [FragmentData] via the [getNavigationIntent] method.
+ */
+class FragmentContainerActivity : BridgeBaseActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_fragment_container)
+
+        val data = getFragmentData(this)
+        setHomeAsUpToolbar(toolbar, data.titleRes)
+
+        if (savedInstanceState != null) return
+
+        val fragment = Fragment.instantiate(
+                this,
+                data.fragmentClass.name,
+                data.fragmentBundle
+        )
+        supportFragmentManager.beginTransaction()
+                .replace(R.id.container, fragment)
```

Comment:
Can you drop `.beginTransaction()` down a line

Reply:
Ah, sure thing.

Reply:
Fixed.